framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -555,34 +558,32 @@ public void run() {
       }
     }
   }
-  
 
-  private static void hdfsCopy(Path src, Path dest, boolean overWrite, String fsMode)
-      throws IOException {
-    LOG.debug("Copy " + src + " to " + dest);
+  private static void dfsCopy(Path src, Path dest, String fsMode)
+          throws IOException {
+
     FileSystem fs;
+    FileSystem localFs = FileSystem.getLocal(conf);
+
     if (fsMode.equals(LOCALFS)) {
       fs = FileSystem.getLocal(conf);
     } else {
       fs = FileSystem.get(conf);
     }
-    FileSystem localFs = FileSystem.getLocal(conf);
-    if (localFs.getFileStatus(src).isDir()) {
-      for (FileStatus file : localFs.listStatus(src)) {
-        Path newSrc = file.getPath();
-        Path newDest = new Path(dest, newSrc.getName());
-        hdfsCopy(file.getPath(), newDest, overWrite, fsMode);
-      }
-    } else if (!fs.exists(dest) || overWrite) {
+
+    if (!fs.exists(dest)) {
```

Comment:
We've never supported overWrite, it was hardcoded to false. So I just removed it. We can check support for overWrite on a later change.